app/templates/userhome.html

```diff
@@ -6,6 +6,8 @@
 
 {% block body %}
 <div class="content">
+
+  <!-- Project -->
```

Comment:
TBH, I'm not a fan of these kinds of comments.

Reply:
okok, ill remove em